cli/repl.rs

```diff
@@ -89,8 +89,7 @@ pub async fn run(
   global_state: &GlobalState,
   mut worker: MainWorker,
 ) -> Result<(), AnyError> {
-  // Our inspector is unable to default to the default context id so we have to specify it here.
-  let context_id: u32 = 1;
+  let mut context_id: u64 = 0;
```

Comment:
Is it safe to have `0` here if `executionContextCreated` is not received?

Reply:
Implementation defined but for us 0 is an invalid context identifier, we start at 1 so this is the uninitialized value.

Per spec, executionContextCreated must always be sent for each context after Runtime.enable has been called.

Reply:
Cool, can you add that comment to code?